filters/openpolicyagent/common.go

```diff
@@ -0,0 +1,6 @@
+package openpolicyagent
+
+const (
+	// Headers to be added in the response sent back to the client.
+	OpenPolicyAgentResponseHeadersKey = "open-policy-agent:decision-response-headers"
```

Comment:
Please put it into "openpolicyagent.go", we do not want to have "common.go" as file.

Reply:
Addressed.